user-journeys/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "user-journeys-demo",
+  "version": "1.0.0",
+  "description": "Replay recorded user journeys of your website on Cloud Run jobs.",
+  "main": "runner.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/steren/user-journeys-demo.git"
```

Comment:
You may want to update these references in this new home

Reply:
Good catch, will send a new PR